### ISSUE TITLE
IntervalVar Java 

### ISSUE THREAD

**Issue by PedroJoaoRibeiro** [google/or-tools#975]
Hello,

I'm trying to add a lunch break interval in JAVA but i don't know how to cast the `IntervalVar[]` to be `com.google.ortools.constraintsolver.SWIGTYPE_p_std__vectorT_operations_research__IntervalVar_p_t`

This is what i'm doing and bellow is the error i get
```java
IntervalVar[] lunchBreakArray = 
  _routingModel.solver().makeFixedDurationIntervalVarArray(
    1,
    n.getStartLunch(),
    n.getEndLunch(),
    n.getDurationbreak(),
    false,
    "lunchBreak");
timeDimension.SetBreakIntervalsOfVehicle(lunchBreakArray, i);
```
```shell
Error:(94, 58) java: incompatible types: com.google.ortools.constraintsolver.IntervalVar[] cannot be converted to com.google.ortools.constraintsolver.SWIGTYPE_p_std__vectorT_operations_research__IntervalVar_p_t
```
Can someone help me?
Ty

**Comment by lperron** [comment]
I guess SetBreaksIntervalsOfVehicle is not correctly wrapped.

I will fix it
Laurent Perron | Operations Research | lperron@google.com | (33) 1 42 68 53
00



Le jeu. 13 déc. 2018 à 13:28, PedroJoaoRibeiro <notifications@github.com> a
écrit :

**Comment by Mizux** [comment]
also notice that `SetBreaksIntervalsOfVehicle` take three parameters...
https://github.com/google/or-tools/blob/c52e9d95b94c9af5cfde91542567ec2e6cbd0757/ortools/constraint_solver/routing.h#L1740-L1751

cpp examples:
https://github.com/google/or-tools/blob/c52e9d95b94c9af5cfde91542567ec2e6cbd0757/examples/cpp/cvrptw_with_breaks.cc#L154-L190

**Comment by Mizux** [comment]
Should be fixed by #1096
